source/sysadmins-guide/varnish-setup/index.md

```diff
@@ -242,10 +242,8 @@ sub vcl_recv {
         return (pass);
     }
 
-    # Do not cache these paths.
-    if (req.url ~ "^/backend" ||
-        req.url ~ "^/backend/.*$") {
-
+    # Always pass these paths directly to php without caching
```

Comment:
Please add a notice that the condition does not work with virtual urls and has been to be changed

Reply:
I assume you are referring to [this](https://developers.shopware.com/developers-guide/controller/#controller-and-urls)? So my suggestion would not work for e.g. `/?module=frontend&controller=checkout&action=index` or another equivalent like `/frontend/checkout/index`. Is that what you meant?

Reply:
Sorry for the late response :)

No if you have virtual urls like /en/checkout /fr/checkout. The condition that you currently have would not work. So a simple note about that would be nice :)

Reply:
No Problem :)
I've added a note in the template.